### PR TITLE
Added Wishlist&cart

### PR DESCRIPTION
**Pull request by sainath7878**
Added Cart and wishlist page to my E-Commerce site
Please find the snaps of the pages:
1. WishList
Screenshot 2022-02-14 at 3 41 41 PM
2. Cart items 
Screenshot 2022-02-14 at 3 45 38 PM

### CODE REVIEW

File: docs/cart/cart.css [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    justify-items: center;
+    grid-template-areas: 
+    "cart-items cart-details";
+    position: relative;
+}
+
+.cart-items{    
+    grid-area: cart-items;
+    padding: 3rem;
+    border-right: 2px solid var(--SIDE-NAVBAR-COLOR);
```
Comment (Dev-AkshatGupta) [inline docs/cart/cart.css]: Better color naming is possible

File: docs/cart/cart.css [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    color: var(--SECONDARY-COLOR);
+}
+
+.quantity{
+    gap: 2rem;
+}
+
+.quantity > input[type="number"]{
+    width: 7.5rem;
+    padding: 1rem
```
Comment (Dev-AkshatGupta) [inline docs/cart/cart.css]: You missed the `;`

Comment (sainath7878) [reply docs/cart/cart.css]: Added ;

File: docs/cart/cart.css (hunk 1 above)
Comment (sainath7878) [reply docs/cart/cart.css]: I'm using the same color as side navbar color that is defined in my root. So, should I change the variable name to some generic name or create a new variable but with same color?

Comment (Dev-AkshatGupta) [reply docs/cart/cart.css]: Do one thing change it here separately  as of now and rename it later on there so for now you will be able to merge it

**Review by Dev-AkshatGupta — APPROVED** [review]

**Review by Maianki — APPROVED** [review]
Looks good to merge.

File: docs/cart/cart.css [hunk 3]
```diff
@@ -0,0 +1,64 @@
+:root{
+    --BORDER-COLOR: #e5e7eb;
+}
+
+.cart{
+    display: grid;
+    grid-template-columns: 60% 40%;
```
Comment (animeshgarai09) [inline docs/cart/cart.css]: Use fr units here

**Review by animeshgarai09 — APPROVED** [review]
Looks good 👍🏻

**Pull request merged**